src/csharp/Microsoft.Spark/RDD/WorkerFunction.cs

```diff
@@ -37,20 +37,21 @@ public WorkerFunction(ExecuteDelegate func)
             WorkerFunction outerFunction)
         {
             return new WorkerFunction(
-                new WrokerFuncChainHelper(
```

Comment:
This is a breaking change right?

Reply:
Oh is it ?  I only renamed it because there was a typo in the name of this method.

Reply:
I don't we can use the previous worker b/c of this, but it may be ok b/c this was only for RDD. Can you update the compatibility scenario in the description and you may need to update the internal worker version as well?